### PR TITLE
Fix attention mask

### PR DESCRIPTION
**Pull request by Chriskuei**

### CODE REVIEW

File: matchzoo/modules/attention.py [hunk 1]
```diff
@@ -19,22 +19,21 @@ class Attention(nn.Module):
...
 
-    def __init__(self, input_size: int = 100, mask: int = 0):
+    def __init__(self, input_size: int = 100):
         """Attention constructor."""
         super().__init__()
         self.linear = nn.Linear(input_size, 1, bias=False)
-        self.mask = mask
 
-    def forward(self, x):
+    def forward(self, x, mask):
```
Comment (caiyinqiong) [inline matchzoo/modules/attention.py]: rename `mask` to `x_mask`
